ksql-functional-tests/src/test/resources/rest-query-validation-tests/insert-values.json

```
@@ -66,6 +66,138 @@
         {"topic": "test_topic", "key": "10", "value": {"ID": null}}
       ]
     },
+    {
+      "name": "rowkey should be set when stream has int key and only id specified in insert",
```

```suggestion
      "name": "rowkey should be set when stream has int key and only key field specified in insert",
```

What's important here is not that a field named `id` is set, but that the key field is set.

Same for other tests.